.github/workflows/performance-benchmarks-kurobako.yml

```diff
@@ -0,0 +1,144 @@
+name: Performance Benchmarks with kurobako
+
+on:
+  workflow_dispatch:
```

Comment:
ref: https://docs.github.com/ja/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch and https://docs.github.com/ja/actions/managing-workflow-runs/manually-running-a-workflow.

Reply:
👍